.github/workflows/uberjar.yml

```diff
@@ -122,6 +128,31 @@ jobs:
     - name: Wait for Metabase to start and reach 100% health
       run: while ! curl -s 'http://localhost:3000/api/health' | grep '{"status":"ok"}'; do sleep 1; done
       timeout-minutes: 3
+
+      # Build experimental ubuntu-based images only for master
+    - name: Build ${{ matrix.edition }} Ubuntu based multi-arch container
```

Comment:
The job `containerize_test_and_push_container` is already longer and hard to decipher. Adding these steps is going to further reduce this job's readability.

Suggestion: create a new job, something like `containerize_multi_arch` or something along that line, and pack these steps into the said job instead.

Bonus: if something is wrong with the `containerize_multi_arch`, it does not hamper the progress of ``containerize_test_and_push_container` and this specific failed job can be rerun independently.

Reply:
@ariya Good idea! Reimplemented as a separate job.